### PR TITLE
ENG-1253 implement tutorial

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by jeffgo10 — CHANGES_REQUESTED** [review]
only 1 comment from me but hopefully others can also review this since this is a massive PR x.x

File: src/state/store.js
Comment (ichalagashvili) [reply src/state/store.js:56]: yes but I can remove it, it was added because I wanted to trace something during debugging :) Forgot to remove..

Comment (jeffgo10) [reply src/state/store.js:56]: okay. I think, because for production purposes, we'll have to remove it. but its good to know there's this option though

**Review by jeffgo10 — APPROVED** [review]
looks good

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/870#pullrequestreview-512594525" title="Approved">jeffgo10</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [jeffgo10]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
